go/libkb/passphrase_login.go

```diff
@@ -251,8 +245,31 @@ func PassphraseLoginPromptThenSecretStore(m MetaContext, usernameOrEmail string,
 	return nil
 }
 
-func StoreSecretAfterLoginWithLKS(m MetaContext, n NormalizedUsername, lks *LKSec) (err error) {
-	return StoreSecretAfterLoginWithLKSWithOptions(m, n, lks, nil)
+func LoadAdvisorySecretStoreOptionsFromRemote(mctx MetaContext) (options SecretStoreOptions) {
+	options = DefaultSecretStoreOptions()
+
+	var ret struct {
+		AppStatusEmbed
+		RandomPW bool `json:"random_pw"`
+	}
+	err := mctx.G().API.GetDecode(mctx, APIArg{
+		Endpoint:    "user/has_random_pw",
```

Comment:
there is a libkb function that calls this but also does cache. I don't know if it would work here though